eng/release/DiagnosticsReleaseTool/Common/NugetLayoutWorker.cs

```diff
@@ -5,7 +5,7 @@ namespace ReleaseTool.Core
     public sealed class NugetLayoutWorker : PassThroughLayoutWorker
     {
         public NugetLayoutWorker(string stagingPath) : base(
-            shouldHandleFileFunc: static file => file.Extension == ".nupkg" && !file.Name.EndsWith(".symbols.nupkg"),
+            shouldHandleFileFunc: static file => file.Extension == ".nupkg" && file.Directory.Name == "packages",
```

Comment:
Two things:
- Can you make the packages folder a constant in `DarcHelpers.cs`? I try to keep all the things where we depend on darc's layout there just for when they change it.
- Also, given that this is the common shared one (I don't have an overly strong stance here) - I think it's better to have the constructor in the runner pass down the override. This is tying it closely to darc, which changes things around in layouts. Something like 
```c#
new NugetLayoutWorker(
  stagingPath: releaseConfig.StagingDirectory.FullName,
  shouldHandleFileFunc: static file => file.Extension == ".nupkg" && file.Directory.Name == DarcHelpers.PackageDirName
),
```